### PR TITLE
fix(validator-types): type Alignment with Web Standards

### PR DESCRIPTION
**Pull request by EdamAme-x**
Close https://github.com/honojs/hono/issues/3119

### The author should do the following, if applicable

- [x] Add tests
- [x] Run tests
- [x] `bun run format:fix && bun run lint:fix` to format the code
- [ ] Add [TSDoc](https://tsdoc.org/)/[JSDoc](https://jsdoc.app/about-getting-started) to document the code

### CODE REVIEW

**Comment by EdamAme-x** [comment]
Check the issue for more information on this issue.
This change brings the types more in line with web standards.

File: src/types.ts [hunk 1]
```diff
@@ -1927,11 +1927,12 @@ type MergeTypedResponse<T> = T extends Promise<infer T2>
 //////                             /////
 ////////////////////////////////////////
 
-export type FormValue = string | File
+export type FormValue = string | File | Blob
```
Comment (yusukebe) [inline src/types.ts:1930]: I think `File` is not necessary:

```ts
export type FormValue = string | Blob
```

**Comment by yusukebe** [comment]
Hey @EdamAme-x 

Thank you for the PR. I've left the comment.

File: src/types.ts (hunk 1 above)
Comment (EdamAme-x) [reply src/types.ts]: Thanks and that's right!
https://www.typescriptlang.org/play/?#code/MYewdgzgLgBAZgSwDYFMYF4ZhQdxgMWRQAoBtAIgAtwRyBdAGhipoDooAPKcgSgG4AUKEiw4IAE4BbACIBDKLIxZcBCTPmzi-AWKlyFrCCijFykgJ6JU5JlZT8gA

**Comment by EdamAme-x** [comment]
@yusukebe I just ready now

**Review by yusukebe — APPROVED** [review]
LGTM!

**Comment by yusukebe** [comment]
@EdamAme-x Thanks. Merging now.

**Pull request merged**